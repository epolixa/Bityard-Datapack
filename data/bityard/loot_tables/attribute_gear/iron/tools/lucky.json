{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:iron_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Iron Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 5.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Iron Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Iron Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.1,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 8.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Iron Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 3.5,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Iron Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}