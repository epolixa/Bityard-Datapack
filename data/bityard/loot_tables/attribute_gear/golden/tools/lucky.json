{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Golden Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Golden Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Golden Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Golden Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.5,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Lucky Golden Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.luck",
                  "name": "Luck",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}